integrations/uphold/src/main/java/org/dash/wallet/integrations/uphold/data/UpholdCardAddressList.kt

```diff
@@ -0,0 +1,17 @@
+package org.dash.wallet.integrations.uphold.data
+
+import com.google.gson.annotations.SerializedName
+
+class UpholdCardAddressList(
+    @SerializedName("formats")
+    val addresses: List<UpholdCardAddress2>,
+    val type: String
+) {
+    fun isEmpty() = addresses.isEmpty()
+    fun firstOrNull() = addresses.firstOrNull()
+}
+
+data class UpholdCardAddress2(
+    val format: String,
+    val value: String
+)
```

Comment:
This needs a different name.  This is a different structure compared to the other `UpholdCardAddress`.